sources/packages/web/src/services/http/ApplicationExceptionApi.ts

```diff
@@ -44,9 +44,9 @@ export class ApplicationExceptionApi extends HttpBaseClient {
   }
 
   /**
-   * Gets all pending student application exceptions.
+   * Gets all pending student application appeals.
```

Comment:
I did not get this change. Can you please elaborate on that?

Reply:
I will revert. I must have touched that by mistake.